src/test/java/org/sonar/plugins/findbugs/FbContribRulesDefinitionTest.java

```diff
@@ -31,18 +33,29 @@
 
 public class FbContribRulesDefinitionTest {
 
-  @Test
-  public void test() {
+  private RulesDefinition.Repository repository;
+
+  @Before
+  public void setUp() {
     FbContribRulesDefinition definition = new FbContribRulesDefinition();
     RulesDefinition.Context context = new RulesDefinition.Context();
     definition.define(context);
-    RulesDefinition.Repository repository = context.repository(FbContribRulesDefinition.REPOSITORY_KEY);
+    repository = context.repository(FbContribRulesDefinition.REPOSITORY_KEY);
+  }
 
+  @Test
+  public void nameIsValid() {
     assertThat(repository.name()).isEqualTo(FbContribRulesDefinition.REPOSITORY_NAME);
+  }
+
+  public void languageIsValid() {
     assertThat(repository.language()).isEqualTo(Java.KEY);
+  }
 
+  @Test
+  public void test() {
     List<Rule> rules = repository.rules();
-    assertThat(rules).hasSize(258);
```

Comment:
I don't think it is a good idea to check an exact number of rules. It's a kind of magic number
To update version you have to do too much things by hand. IMO checking if rule set is not empty is enough